generate.py

```diff
@@ -411,7 +411,31 @@ def generate_patch_bay_labels_from_json(config, index):
             "normalled": False,
             "top": "-",
             "bottom": "-",
-            "width": 16
+            "width": 12
+        },
+        {
+            "normalled": False,
+            "top": "Audio In",
+            "bottom": "Mix Out",
+            "width": 1
+        },
+        {
+            "normalled": True,
+            "top": "Loop Out",
+            "bottom": "Loop In",
+            "width": 1
+        },
+        {
+            "normalled": False,
+            "top": "Delay Out",
+            "bottom": "Feedback",
+            "width": 1
+        },
+        {
+            "normalled": False,
```

Comment:
feedback, mix and time are CV inputs. If this is being connected to a modulation pedal, do we need to use an insert / TRS cable and present it as two patch points?

Reply:
related to #12